content/en/docs/concepts/policy/_index.md

```diff
@@ -4,3 +4,8 @@ weight: 90
 description: >
   Policies you can configure that apply to groups of resources.
 ---
+
+{{< note >}}
+See [Network Policies](/docs/concepts/services-networking/network-policies/)
+for documentation about NetworkPolicy in Kubernetes.
```

Comment:
Hi @sftim. Is there a good reason to include a link to the Network Policies section from this index page? Why does this information need to be in a note?

Reply:
If someone is looking for network policies, they might look for it in the Policies section, but it's actually in the Networking section. This note is to help people who looked in Policies find the thing that they are looking for.

Maybe we have a different mechanism to add it into the navigation itself; if we do, I don't know about that other section.